custom_components/tuya_local/devices/plikc_neve_pro_thermostat.yaml

```diff
@@ -0,0 +1,176 @@
+name: Thermostat
+products:
+  - id: 9myqvomgygt9zgcs
+    name: Plikc Neve Pro W
+primary_entity:
+  entity: climate
+  translation_key: thermostat
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          value: heat_cool
+    - id: 2
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: auto
+          value: program
+        - dps_val: manual
+          value: manual
+        - dps_val: temporary
+          value: temp_override
+          hidden: true
+        - dps_val: holiday
+          value: away
+          constraint: holiday_duration
+          conditions:
+            - dps_val: 0
+              value: "Set holiday length!"
+              hidden: true
+    - id: 3
+      type: string
+      name: hvac_action
+      mapping:
+        - dps_val: heat
+          value: heating
+        - dps_val: cold
+          value: cooling
+        - dps_val: "off"
+          constraint: hvac_mode
+          conditions:
+            - dps_val: true
+              value: idle
+            - dps_val: false
+              value: "off"
+    - id: 16
+      type: integer
+      name: temperature
+      unit: C
+      range:
+        min: 5
+        max: 350
+      mapping:
+        - scale: 1
+          constraint: preset_mode
+          conditions:
+            - dps_val: holiday
+              value_redirect: holiday_temp_set
+    - id: 19
+      type: integer
+      name: upper_temperature
+      unit: C
+      range:
+        min: 50
+        max: 350
+      mapping:
+        - scale: 1
+          constraint: preset_mode
+          conditions:
+            - dps_val: holiday
+              value_redirect: holiday_temp_set
+    - id: 24
+      type: integer
+      name: current_temperature
+      mapping:
+        - scale: 1
+    - id: 26
+      type: integer
+      name: lower_temperature
```

Comment:
I think this should be `min_temperature`. Otherwise, same comments as above apply.